tests/PriorityMapTest.php

```diff
@@ -144,4 +157,31 @@ public function testSet_ObjectKey()
         $list->rewind();
         $this->assertEquals(41, $list->current());
     }
+
+    public function testSetDescOrder()
+    {
+        $key = new \stdClass();
+
+        $list = new PriorityMap();
+        $list->setDescOrder();
+
+        $list->set($key, 42, 1);
+        $list->set($key, 41, 0);
+        $list->set($key, 43, 2);
+
+        $this->assertSame(43, $list->current());
+    }
+
+    public function testToArray()
+    {
+        $key = new \stdClass();
+
+        $list = new PriorityMap();
+
+        $list->set($key, 42, 1);
+        $list->set($key, 41, 0);
+        $list->set($key, 43, 2);
+
+        $this->assertInternalType('array', $list->toArray());
```

Comment:
if method will return `[100500]` you test will pass but code fails. this test test nothing.